app/models/concerns/globalized.rb

```diff
@@ -18,6 +18,34 @@ def translates(*columns)
       super(*columns, fallbacks_for_empty_translations: true)
     end
 
+    def translates_rich_text(*columns)
```

Comment:
Kannst du hier in der commit message noch genauer ausführen wieso dieser Tanz notwendig ist? In diesem Fall fände ich ein Kommentar im Code und Details in der Commit Message hilfreich. Beziehst du dich auf https://github.com/rails/actiontext/issues/32#issuecomment-450653800?